src/main.rs

```diff
@@ -329,11 +333,16 @@ fn ca_chain_builtin(chain: &sev::Chain) -> Result<ca::Chain> {
 }
 
 fn main() {
+    env_logger::init();
+
     let sevctl = Sevctl::from_args();
     let status = match sevctl.cmd {
         SevctlCmd::Export { full, destination } => export::cmd(full, destination),
         SevctlCmd::Generate { cert, key } => generate::cmd(cert, key),
         SevctlCmd::Ok { gen } => ok::cmd(gen, sevctl.quiet),
+        SevctlCmd::Measurement(option) => match option {
+            measurement::MeasurementCmd::Build(args) => measurement::build_cmd(args),
+        },
```

Comment:
Is this match here in anticipation in more `sevctl measurement` commands in the future?

Reply:
I don't have any in mind. I was following the `vmsa` subcommand examples.